src/util/http_utils.js

```diff
@@ -318,6 +318,8 @@ async function parse_request_body(req, options) {
 }
 
 function send_reply(req, res, reply, options) {
+    res.setHeader('Server', 'NooBaa');
```

Comment:
1. I think this should be called at the beginning of the request, not at the end, so that it also applies to errors or any other forms of replying, so I would move it to endpoint.js or maybe add it inside the existing calls to endpoint_utils.prepare_rest_request().
2. I think we should return also the server version - like `Server: Apache/2.4.1 (Unix)` which you can get from `const pkg = require('../../package.json'); ... pkg.version`